### PR TITLE
Fixes from integration tests

### PR DESCRIPTION
**Pull request by juanjux**
Several fixes for problems found with the python-driver integation tests.

### CODE REVIEW

File: pydetector/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.10.0"
+__version__ = "0.10.3"
```
Comment (abeaumont) [inline pydetector/version.py]: why the jump from 0 to 3?

Comment (juanjux) [inline pydetector/version.py]: Oh yes, forgot to comment on this; this is the proof that I need to change the Dockerfile.build.tpl to copy and pip-install pydetector-bblfsh from the local filesystem without using Pypi because if not the only way to test is to publish releases. Will do on another PR soon for this.

Comment (juanjux) [inline pydetector/version.py]: PS: https://github.com/bblfsh/python-driver/issues/41
